packages/nocodb/src/lib/db/sql-data-mapper/lib/sql/BaseModelSqlv2.ts

```diff
@@ -1808,7 +1819,19 @@ class BaseModelSqlv2 {
         }
 
         if (ai) {
-          // response = await this.readByPk(id)
+          if (this.isSqlite) {
+            // sqlite doesnt return id after insert
+            id = (
+              await this.dbDriver(this.tnPath)
+                .select(ai.column_name)
+                .max(ai.column_name, { as: 'id' })
+            )[0].id;
```

Comment:
Are you sure that SQLite doesn't return the autoincremented primary key on insert?

Reply:
Looks like knex introduced returning for sqlite3 in [v2.0.3](https://github.com/knex/knex/releases/tag/2.3.0)
I will cover this on another PR as this is a little bit out of scope